.github/workflows/buildifier.yaml

```diff
@@ -1,16 +1,13 @@
 name: Buildifier
-
```

Comment:
yaml formatter tool out the whitespace?

Reply:
Yep, that’s what prettier runs on by default with rules lint apparently. Maybe we should file a rules lint bug?

Reply:
Indeed. I rebased this and tried running `bazel run //:format` and it destroys all of the whitespace in the config yaml even tho prettier is configured for JavaScript only: `javascript = ":prettier",`.

@alexeagle Do you know what is going on here?

Reply:
Hmm. This PR only has the js formatter configured:

```
prettier.prettier_binary(
    name = "prettier",
    # Allow the binary to be run outside bazel
    env = {"BAZEL_BINDIR": "."},
)

format_multirun(
    name = "format",
    javascript = ":prettier",
)
```

but it is formatting JavaScript, JSON, TypeScript, Shell & YAML.... are those other formatters unintentionally on by default or is prettier formatting them?

Here is from my local command line:

```
$ git status
On branch mrmeku-rules_lint
Your branch is up to date with 'origin/mrmeku-rules_lint'.

$ bazel run //:format
INFO: Analyzed target //:format (96 packages loaded, 5352 targets configured).
INFO: Found 1 target...
Target //tools/format:format up-to-date:
  bazel-bin/tools/format/format.bash
INFO: Elapsed time: 0.410s, Critical Path: 0.00s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Running command line: bazel-bin/tools/format/format.bash
e2e/workspace/module_test/cypress.config.js 23ms (unchanged)
e2e/workspace/module_test/module_test.cy.js 1ms (unchanged)
e2e/workspace/module_test/runner.js 3ms (unchanged)
e2e/workspace/server_example/cypress.config.js 7ms (unchanged)
e2e/workspace/server_example/server.js 2ms (unchanged)
e2e/workspace/server_example/server_example_test.cy.js 1ms (unchanged)
Formatted JavaScript in 0m0.114s
.bcr/metadata.template.json 17ms (unchanged)
.bcr/source.template.json 0ms (unchanged)
e2e/workspace/cli_test/tsconfig.json 1ms (unchanged)
e2e/workspace/package.json 1ms (unchanged)
package.json 0ms (unchanged)
renovate.json 1ms (unchanged)
Formatted JSON in 0m0.084s
e2e/workspace/cli_test/cli_test.cy.ts 81ms (unchanged)
e2e/workspace/cli_test/cypress.config.ts 6ms (unchanged)
Formatted TypeScript in 0m0.161s
Formatted Shell in 0m0.001s
Formatted YAML in 0m0.007s

$ git status
On branch mrmeku-rules_lint
Your branch is up to date with 'origin/mrmeku-rules_lint'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .github/workflows/buildifier.yaml
	modified:   .github/workflows/ci.yaml
	modified:   .github/workflows/release.yml
	modified:   .pre-commit-config.yaml
	modified:   e2e/workspace/pnpm-lock.yaml
	modified:   pnpm-lock.yaml
```

[Prettier 1.14: YAML Support](https://prettier.io/blog/2018/07/29/1.14.0)

Reply:
Using rules_lint 0.19.0 in this PR:
```
bazel_dep(name = "aspect_rules_lint", version = "0.19.0", dev_dependency = True)
```

Reply:
rules_lint requires explicit opt-out for tools that are builtin.

`yaml = None` should turn off the formatting